src/psyclone/tests/psyad/main_test.py

```diff
@@ -512,3 +566,47 @@ def test_main_otest_verbose(tmpdir, caplog):
                      "github actions.")
     assert "Writing test harness for adjoint kernel to file" in caplog.text
     assert "/harness.f90" in caplog.text
+
+
+def test_main_otest_lfric(tmpdir, capsys, caplog):
```

Comment:
I don't believe there was something that tested running `main.py` with the LFRic API. Previously this was fine because `main.py` was not modified but with the new changes to it I wanted to make sure that the behaviour was working correctly when given a filename. Perfectly fine to remove it if it is superfluous!

Reply:
Ah, OK. Makes sense thanks.